src/Nethermind/Nethermind.Network/SessionMonitor.cs

```diff
@@ -150,14 +138,18 @@ private void StartPingTimer()
                 try
                 {
                     _pingTimer.Enabled = false;
-                    SendPingMessages();
+                    SendPingMessagesAsync().ContinueWith(x =>
+                    {
+                        if (x.IsFaulted && _logger.IsError)
+                        {
+                            _logger.Error($"DEBUG/ERROR Error during send ping messages: {x.Exception}");
+                        }
+                        _pingTimer.Enabled = true;
```

Comment:
Not neede? it is in finally

Reply:
Removed from finally (and added to catch) because this bit will complete after the method returns and timer thread returns; so needs to be switched back on at completion.